pagopa/pn-frontend
# PN-5484 - add copy to analog flow events with legal facts attached

## Short description
As an addenda to PN-5484, it was decided to show "C'è un nuovo documento allegato." as copy for the timeline events (mostly SEND_ANALOG_PROGRESS) having attached a legal fact.

## List of changes proposed in this pull request
- Just modified the notifiche.json accordingly.

## How to test
Enter an analog flow notification in PA / PF / PG, open detail for the status "Invio in corso". The events including an attachment just have the indicated copy.  
Example: ambiente DEV, IUN VZWU-ULXW-TRMG-202305-Y-1, Comune di Palermo (the recipient is Maria Montessori, but she hasn't accepted the TOS and it should be kept that way). For PF, most probably cristoforocolombo will have several analog flow notifications (you can see "analogico" in the description).
